eggo/dag.py

```diff
@@ -226,33 +242,38 @@ def run(self):
         p = Popen(delete_raw_cmd, shell=True)
         p.wait()
 
-class VCF2ADAMTask(Task):
+class ADAMBasicTask(Task):
 
     config = ConfigParameter()
+    adam_command = Parameter()
+    allowed_file_formats = Parameter()
+    edition = 'basic'
 
     def _raw_data_s3_url(self):
```

Comment:
can we get rid of these methods entirely and just use the module-level functions?